clippy_lints/Cargo.toml

```diff
@@ -25,13 +25,14 @@ matches = "0.1.7"
 pulldown-cmark = "0.6.0"
 quine-mc_cluskey = "0.2.2"
 regex-syntax = "0.6"
+semver = "0.9.0"
 serde = { version = "1.0", features = ["derive"] }
 smallvec = { version = "1", features = ["union"] }
+syn = { version = "1.0", features = ["full"] }
 toml = "0.5.3"
 unicode-normalization = "0.1"
-semver = "0.9.0"
-# NOTE: cargo requires serde feat in its url dep
-# see <https://github.com/rust-lang/rust/pull/63587#issuecomment-522343864>
+# cargo requires serde feat in its url dep
+# see https://github.com/rust-lang/rust/pull/63587#issuecomment-522343864
 url = { version =  "2.1.0", features = ["serde"] }
```

Comment:
I think `features` could be dropped entirely because it's handled by `rustc-workspace-hack`: https://github.com/rust-lang/rust/blob/4752c05af4a5f392de026f9ae1877eae030be359/src/tools/rustc-workspace-hack/Cargo.toml#L66

Dogfood has hidden the error but maybe this change will help (it failed to build `url`).